core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -92,6 +92,20 @@ private MetadataColumns() {}
           Types.LongType.get(),
           "Commit snapshot ID");
 
+  // the content size and offset of a DV
```

Comment:
I wouldn't necessarily bind these fields to DVs. They can make sense for other use cases in the future. Also, they do seem more like metadata columns to me, rather than reserved columns. This class defines both.

The `position_deletes` metadata table uses `FILE_PATH_COLUMN_ID`, which is a metadata column. I'd probably define these field IDs as metadata and just give names to them in `PositionDeletesTable`. I see the content size and offset used together with the file name records come from.

Reply:
I've addressed this in https://github.com/apache/iceberg/pull/11808